lib/tasks/backfill_in_person_pending_at.rake

```diff
@@ -75,4 +75,67 @@ namespace :profiles do
 
     warn "backfill_in_person_verification_pending_at left #{profiles.count} rows"
   end
+
+  ##
+  # Usage:
+  #
+  # Print pending updates
+  # bundle exec rake profiles:backfill_deactivated_ipp_verification_cancelled
+  #
+  # Commit updates
+  # bundle exec rake profiles:backfill_deactivated_ipp_verification_cancelled UPDATE_PROFILES=true
+  #
+  task backfill_deactivated_ipp_verification_cancelled: :environment do |_task, _args|
+    ActiveRecord::Base.connection.execute('SET statement_timeout = 60000')
+
+    update_profiles = ENV['UPDATE_PROFILES'] == 'true'
+
+    InPersonEnrollment.where(status: [:expired, :cancelled, :failed]).
+      includes(:profile).
+      where.not(profile: { in_person_verification_pending_at: nil }).
+      find_in_batches do |batch|
+        batch.each do |enrollment|
+          profile = enrollment.profile
+          timestamp = profile.in_person_verification_pending_at
+
+          warn "#{profile.id},#{profile.deactivation_reason},#{timestamp}"
+          if update_profiles
```

Comment:
TBH I wouldn't change it at this point, but I feel like it would be a tiny bit more ideal if the `warn` on line 101 occurred inside this conditional, or there was otherwise some indication if `update_profiles` was false. It'd otherwise be easy to miss that this script didn't change anything if the environment variable wasn't set.

_But_, it looks like this has already been extensively tested, so it doesn't feel like it's worth changing. I'd just make sure someone confirms the script was successful, which was probably part of the plan all along anyway. 👍

Reply:
Good callout! I did add checks in the roll plan for verifying the data.

Reply:
I thought that you put the warn outside of the if so you could do a test run to ensure you had access to all of the data before running it. (To ensure you had data for rollback if needed)